package.json

```diff
@@ -17,7 +17,7 @@
   "author": "Brian M. Carlson",
   "license": "MIT",
   "dependencies": {
-    "nan": "1.5.0",
+    "nan": "1.5.3",
```

Comment:
Is there a strong reason for not using "^1.7.0" ?

I doubt that the nan module will break anything as the minor or patch version numbers change.  And my understanding of the reason for the nan module's existence is so that you can easily incorporate updates that come in as a result of changes in V8.  If we lock libpq's nan dependency to a specific version we won't get needed updates as V8 changes come down along (among other things).